lib/bn.js

```diff
@@ -731,6 +731,41 @@ BN.prototype.uxor = function uxor(num) {
 };
 
 
+// Not ``this`` with ``width`` bitwidth
+BN.prototype.inotn = function inotn(width) {
+  assert(typeof width === 'number' && width >= 0);
+
+  var bytesNeeded = Math.ceil(width / 26) | 0;
+  var bitsLeft = width;
+
+  // Extend the buffer with leading zeroes
+  while (this.length < bytesNeeded) {
+    this.words[this.length++] = 0;
+  }
+
+  // Handle complete words
+  for (var i = 0; i < bytesNeeded && bitsLeft >= 26; i++, bitsLeft -= 26) {
```

Comment:
I think `i < bytesNeeded` and `bitsLeft >= 26` are the same thing, doesn't look like we need both of them. Or am I wrong?